gulpfile.babel.js

```diff
@@ -7,10 +7,12 @@ import gulp from 'gulp';
 import gulpIf from 'gulp-if';
 import mochaRunCreator from './test/mochaRunCreator';
 import path from 'path';
+import realFavicon from 'gulp-real-favicon';
 import runSequence from 'run-sequence';
 import shell from 'gulp-shell';
 import webpackBuild from './webpack/build';
 import yargs from 'yargs';
+import { injectFavicon } from './favicon/favicon';
```

Comment:
I think at this point we need a special dir for qulp tasks. favicon belongs there. /gulp